dali/operators/nvcvop/nvcvop.cc

```diff
@@ -153,12 +153,14 @@ void AllocateImagesLike(nvcv::ImageBatchVarShape &output, const TensorList<GPUBa
 
 void PushImagesToBatch(nvcv::ImageBatchVarShape &batch, const TensorList<GPUBackend> &t_list) {
   auto channel_dim = t_list.GetLayout().find('C');
+  auto num_channels = (channel_dim >= 0) ? t_list[0].shape()[channel_dim] : 1;
```

Comment:
```suggestion
  auto num_channels = (channel_dim >= 0) ? t_list.shape()[[0]channel_dim] : 1;
```
or
```suggestion
  auto num_channels = (channel_dim >= 0) ? t_list.TensorShape(0)]channel_dim] : 1;
```
Much cheaper to extract entry from the shape than to construct a SampleView just to get its shape.